src/Dialect/ONNX/ONNXUnsupportedOps.hpp

```diff
@@ -76,3 +64,17 @@ UNSUPPORTED_OPS(ONNXTreeEnsembleClassifierOp)
 UNSUPPORTED_OPS(ONNXTreeEnsembleRegressorOp)
 UNSUPPORTED_OPS(ONNXUpsampleV7Op)
 UNSUPPORTED_OPS(ONNXZipMapOp)
+// These ops were decomposed into supported ops during Decompose pass.
```

Comment:
That is a great idea to separate the two lists. Do you mind just putting a bit more comments for both? Say

```
// Set of ONNX Ops that we currently do not support, listed alphabetically. Please remove from list when support is added.
...

// Set of ONNX Ops that are supported indirectly as every instances are converted/decomposed in one or more supported ONNX ops. Listed alphabetically. 
#define CONVERTED_TO_SUPPORTED_OPS(_x) UNSUPPORTED_OPS(_x)
#CONVERTED_TO_SUPPORTED_OPS(ONNXCastLikeOp)

...```

Reply:
Thanks! I changed this. It looks better now.